testsuite/pom.xml

```diff
@@ -46,7 +46,7 @@
         <strimzi-oauth.version>1.0.0-SNAPSHOT</strimzi-oauth.version>
         <checkstyle.dir>..</checkstyle.dir>
 
-        <kafka.docker.image>quay.io/strimzi/kafka:0.31.1-kafka-3.2.3</kafka.docker.image>
+        <kafka.docker.image>quay.io/strimzi/kafka:0.33.0-kafka-3.3.1</kafka.docker.image>
```

Comment:
Any reason for using 3.3.1 and not 3.3.2? (both here and below). It probably doesn't matter much, just wondering

Reply:
No reason. I updated to the latest.